ethereum/houses/shibaswap.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Shibaswap",
```

Comment:
Please change `name` to `shibaswap`, all lowercase. Then it's good to go.

Reply:
Hi , ive made the change as requested, i also took the opportunity to add a missing contract